### PR TITLE
feat(ui): trace view network tab add simple filter section

### PR DESCRIPTION
**Pull request by Gigiz**
Proposed feat for #27530 

I added a simple search input to improve the experience of searching for a particular request (filtering by request url).
For the future I'd like to add more ways to filter requests, for example filtering by type (fetch, js, css, doc etc..), like Chrome DevTools or similar

[Open Source Saturday](https://www.meetup.com/it-IT/Open-Source-Saturday-Milano/)

### CODE REVIEW

File: tests/library/trace-viewer.spec.ts [hunk 1]
```diff
@@ -246,6 +246,31 @@ test('should have network requests', async ({ showTraceViewer }) => {
   await expect(traceViewer.networkRequests).toContainText([/200GET\/frames\/script.jsapplication\/javascript/]);
 });
 
+test('should filtering a request in network tab', async ({ showTraceViewer }) => {
```
Comment (Cadienvan) [inline tests/library/trace-viewer.spec.ts:249]:
```suggestion
test('should filter a request in network tab', async ({ showTraceViewer }) => {
```

That's awesome! Just proposed a small grammar correction!

Comment (Gigiz) [reply tests/library/trace-viewer.spec.ts]: modified

**Comment by tonycaputome** [comment]
up!

**Comment by pavelfeldman** [comment]
I applied the change and although it looks promising, PR will require substantial amount of back and forth on the UX work / polish. We try to be efficient and stay away from this model, hence closing.

**Pull request closed without merging**